src/script/cpp_api/s_security.cpp

```diff
@@ -18,7 +18,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 */
 
 #include "cpp_api/s_security.h"
-
+#include "lua_api/l_base.h"
```

Comment:
Is that intentional?

Reply:
The deletion of the newline? I did that so that it matches upstream Minetest: https://github.com/minetest/minetest/blob/644f145ff260c6f9cf0e351d158c045019aa895d/src/script/cpp_api/s_security.cpp#L20-L22

Reply:
Adding `#include "lua_api/l_base.h"`

Reply:
That's needed so that `ModApiBase::getScriptApiBase()` can be called.

Reply:
Can you merge this line change with the first commit? I would like to use cherry-pick instead of the squash & merge.

Reply:
Done.